don't build some mac/win stuff, don't copy third-party libraries (they're not even included in freeplane_srcpure!)
Index: freeplane-1.3.12/freeplane/ant/build.xml
===================================================================
--- freeplane-1.3.12.orig/freeplane/ant/build.xml	2014-09-20 17:36:05.731635226 +0200
+++ freeplane-1.3.12/freeplane/ant/build.xml	2014-09-20 17:36:05.727635226 +0200
@@ -192,14 +192,16 @@
 		</jar>
 	</target>
 	
-	<target name="dist" depends="jar, signjar, signjarinfo, buildmac">
+	<target name="dist" depends="jar, signjar, signjarinfo">
 		<!-- third party libs -->
 		<mkdir dir="${freeplane.dist.lib}" />
+<!--
 		<copy todir="${freeplane.dist.lib}">
 			<fileset dir="${freeplane.ext.lib}">
 				<include name="*.jar" />
 			</fileset>
 		</copy>
+-->
         <mkdir dir="${dist.resources}" />
         <copy todir="${dist.resources}">
             <fileset dir="external-resources"/>
Index: freeplane-1.3.12/freeplane_framework/ant/build.xml
===================================================================
--- freeplane-1.3.12.orig/freeplane_framework/ant/build.xml	2014-09-20 17:36:05.731635226 +0200
+++ freeplane-1.3.12/freeplane_framework/ant/build.xml	2014-09-20 17:36:05.727635226 +0200
@@ -166,10 +166,6 @@
 		<copy todir="${build}">
 			<fileset dir="${script}" />
 			<fileset file="${framework.jar}" />
-			<fileset file="${freeplanelauncher.jar}" />
-			<fileset file="windows-icons/freeplaneIcons.dll" />
-			<fileset dir="launch4j" includes="*.exe" />
-			<fileset dir="launch4j" includes="*.ini" />
 		</copy>
 		<chmod file="${build}/freeplane.sh" perm="ugo+rx" />
 	</target>
